libtock/qdec.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "tock.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define DRIVER_NUM_QDEC 0x90003
+
+bool qdec_exists(void);
+bool qdec_enable(void);
+bool qdec_interrupt_enable(void);
```

Comment:
This may be more of an issue to address on the kernel side, but from a userspace perspective, this is a pretty low-level interface -- how often do interrupts fire (every partial rotation? after some displacement or interval?).

More generally, in the header file here, some documentation on how each of these functions expect to be called / what they expect to return would be helpful.

Reply:
Interrupts fire every partial rotation. Each `tick` of the rotary encoder is equivalent to 4 clicks, and the interrupt comes at each of those clicks. I will make sure to add some helpful documentation for the next round of review.